fluid/face_detction/utility.py

```diff
@@ -0,0 +1,60 @@
+"""Contains common utility functions."""
```

Comment:
Please move this line to the bottom of the license. Or should we remove the license?

Reply:
Done. remove these lines.